cycode/cli/helpers/tf_content_generator.py

```diff
@@ -0,0 +1,29 @@
+import json
+from typing import List
+
+from cycode.cli.models import ChangeResource
+
+
+def generate_tf_content_from_tfplan(tfplan: str) -> str:
+    planned_resources = _extract_resources(tfplan)
+    return _generate_tf_content(planned_resources)
+
+
+def _extract_resources(tfplan: str) -> List[ChangeResource]:
+    tfplan_json = json.loads(tfplan)
```

Comment:
let's create a function that will load json and handle all errors. if errors have occurred the None will be returned. for example check "get_file_content" function